app/controlplane/internal/service/service.go

```diff
@@ -168,6 +168,8 @@ func WithGroupUseCase(groupUseCase *biz.GroupUseCase) NewOpt {
 // For example: `s.authorizeResource(ctx, authz.PolicyAttachedIntegrationDetach, authz.ResourceTypeProject, projectUUID);`
 // checks if the user has a role in the project that allows to detach integrations on it.
 // This method is available to every service that embeds `service`
+// It goes through all the memberships of the user, direct memberships and indirect memberships (Groups)
```

Comment:
@javirln Check also `visibleProjects` method. We might need to add groups memberhips there as well.

Reply:
`entities.CurrentMembership(ctx)` contains the memberships of the user and all of her groups. When looping through the Resources, it already takes into account the groups' roles